### ISSUE TITLE
Latest version of TypeScript causing errors

### ISSUE THREAD

**Issue by TomSmedley** [paulcollett/react-masonry-css#87]
Hello,

Thank you for putting in so much effort into React Masonry, it's one of the best components I've worked with.

Recently after updating my project, I'm getting a TypeScript error:
![image](https://user-images.githubusercontent.com/95056193/189887228-054d5abe-6850-4a42-bd16-268a7e28128f.png)

It doesn't stop the app from working, though it's never nice seeing the error symbol in your IDE.

**Comment by JaredPage** [comment]
Yep I am also experiencing this. Tried a few things like `preseveSymlinks` and type -> `react+jsx` in tsconfig.json to no avail. It seems it's just on TS >= v18.
